### ISSUE TITLE
Example does not work

### ISSUE THREAD

**Issue by slowkow** [likelet/IDEA#7]
I tried running the example, but it didn't work for me.

See below for the steps I tried to follow.

![2018-07-04 at 9 39 am](https://user-images.githubusercontent.com/209714/42280511-f3d146da-7f6e-11e8-9956-3fd9da9b7ed0.png)

![2018-07-04 at 9 41 am](https://user-images.githubusercontent.com/209714/42280515-f6447f36-7f6e-11e8-92c6-c03e460074bc.png)

![2018-07-04 at 9 42 am](https://user-images.githubusercontent.com/209714/42280517-f7bcc922-7f6e-11e8-98cf-15cf5ac42a5d.png)

![2018-07-04 at 9 43 am](https://user-images.githubusercontent.com/209714/42280521-f9322608-7f6e-11e8-94fc-7ebc50c090cd.png)

![2018-07-04 at 9 44 am](https://user-images.githubusercontent.com/209714/42280523-fafc7916-7f6e-11e8-9fed-3232e6e3f742.png)

**Comment by likelet** [comment]
hi slowkow,
Thanks for your concern of IDEA, it seems that DESeq2 have changed some functions in their update version. And perhaps one of  my college modified the R environment of the remote server,  I will check it ASAP.
 Qi

**Comment by likelet** [comment]
hi slowkow,

Sorry for the late reply, it's indeed a bug from the wrong named  variables in `server.R`. I fixed it by adding a rename function in data parsing step. I noticed that you run multi-factor example directly before the example component  finishing loading,  I think some functions may not be loaded properly. I suggest to run multi factor analysis in the following steps :  

* 1, click the standard example first until the ‘ button’  loading completely at the right  

![image](https://user-images.githubusercontent.com/3021506/42556007-c6d97a5e-851c-11e8-9d77-e8adcadacc0f.png)  

* 2, Next, go to the multi factor design example  until it shows a message below

![image](https://user-images.githubusercontent.com/3021506/42556106-155b9f9a-851d-11e8-8269-fc95f592db5e.png). 

* 3, then go ahead like you did  before

I  am sorry for the inconvenient for this operation, actually  I have no idea how to skip the "SC click" at present. All I know is that we have to waiting the server side  load all R functions and dependency before analysis. 

Qi
